source/common/upstream/load_balancer_impl.h

```diff
@@ -224,7 +224,7 @@ class ZoneAwareLoadBalancerBase : public LoadBalancerBase {
   /**
    * Pick the host source to use, doing zone aware routing when the hosts are sufficiently healthy.
```

Comment:
optional: it'd be nice to update the comment about when/wny we don't return a host source